cl/corpus_importer/court_regexes.py

```diff
@@ -272,17 +272,28 @@
 # noinspection PyPep8
 fb_pairs = (
     (re.compile('(^|\s)D(\.|(istrict)) of Columbia', re.I), 'dcb'),
+        (re.compile('(|\s)D(\.|(istrict)) of Columbia', re.I), 'dcb'),
```

Comment:
These regexes make me a bit uncomfortable (as we knew they would). Am I wrong that a lot of them just remove the caret? I wouldn't have expected that to make a difference. Can you take a look at why that might be?

Reply:
The plan was to roll back these changes, right? I imagine that should be pretty easy via git?